pkg/log/zap/zap.go

```diff
@@ -149,6 +137,48 @@ func (o *Options) addDefaults() {
 	o.ZapOpts = append(o.ZapOpts, zap.AddStacktrace(o.StacktraceLevel))
 }
 
+// Development sets the Development option for the logger
+func Development(development bool) func(o *Options) {
```

Comment:
Would it make sense for this to not have the boolean arg, considering it wont do anything if the arg is set to `false`?

Reply:
You're right. This is pretty minor but I'm on the fence about this.
Without the arg it's an implicit assumption on the default being false.
And as the caller you can't be explicit about setting a Production config without the arg. 

Not sure if there is a convention for functional options on this.

Reply:
Actually scratch that. I completely misread what you were saying :sweat_smile:
You're right, but it's a moot point now after https://github.com/kubernetes-sigs/controller-runtime/pull/646#issuecomment-542958563